### PR TITLE
[CLOUD-253][Doc] Enable scale from zero for node groups

### PR DESCRIPTION
**Pull request by esierra-stratio**

### CODE REVIEW

**Comment by stratiocommit** [comment]
This PR has modified files that have not been added for all versions. Please add them including the text [to be translated] so that the reviewer can easily upload or modify the translated versions. Missing folder/files:
EN - modules/ROOT/pages/installation.adoc

**Comment by stratiocommit** [comment]
This PR has modified files that have not been added for all versions. Please add them including the text [to be translated] so that the reviewer can easily upload or modify the translated versions. Missing folder/files:
EN - modules/ROOT/pages/installation.adoc

File: stratio-docs/es/modules/ROOT/pages/installation.adoc [hunk 1]
```diff
@@ -340,7 +340,7 @@ Las imágenes utilizadas deberán estar soportadas por EKS. Consulta la https://
 |t3.medium
 |No
 
-|_max++_++size_/_min++_++size_
+|_max++_++size_/_min++_++size**_
```
Comment (iamjanr) [inline stratio-docs/es/modules/ROOT/pages/installation.adoc:343]: Quizas sustituiría los "**" por una "Nota".

File: stratio-docs/es/modules/ROOT/pages/installation.adoc [hunk 2]
```diff
@@ -393,6 +393,8 @@ root_volume:
 |Sí
 |===
 
+_** Se ha implementado la opción de establecer min_size como cero, lo que permite que el autoscalado pueda incrementar o disminuir el número de nodos hasta alcanzar cero según sea necesario. Esta funcionalidad facilita un ahorro significativo de costos en comparación con versiones anteriores, ya que posibilita la precreación virtual de parte de la infraestructura o su eliminación en el caso de considerarse no necesaria._
+
```
Comment (stg-0) [inline stratio-docs/es/modules/ROOT/pages/installation.adoc:397]:
```suggestion
NOTE: Se ha implementado la opción de establecer un min_size igual a cero, lo que permite que el autoscalado pueda incrementar o disminuir el número de nodos hasta alcanzar cero según sea necesario. Esta funcionalidad facilita un ahorro significativo de costos en comparación con versiones anteriores, ya que posibilita la definición de un grupo de workers sin instanciar ningún recurso en el _cloud provider_ que no sea necesario.

```

File: stratio-docs/es/modules/ROOT/pages/installation.adoc (hunk 1 above)
Comment (stg-0) [inline stratio-docs/es/modules/ROOT/pages/installation.adoc:343]:
```suggestion
|_max++_++size_/_min++_++size_
```

**Comment by stratiocommit** [comment]
This PR has modified files that have not been added for all versions. Please add them including the text [to be translated] so that the reviewer can easily upload or modify the translated versions. Missing folder/files:
EN - modules/ROOT/pages/installation.adoc

**Review by stg-0 — APPROVED** [review]
LGTM

**Comment by stratiocommit** [comment]
This PR has modified files that have not been added for all versions. Please add them including the text [to be translated] so that the reviewer can easily upload or modify the translated versions. Missing folder/files:
EN - modules/ROOT/pages/installation.adoc

File: stratio-docs/es/modules/ROOT/pages/installation.adoc [hunk 3]
```diff
@@ -393,6 +393,8 @@ root_volume:
 |Sí
 |===
 
+NOTE: Se ha implementado la opción de establecer un min_size igual a cero, lo que permite que el autoscalado pueda incrementar o disminuir el número de nodos hasta alcanzar cero según sea necesario. Esta funcionalidad facilita un ahorro significativo de costos en comparación con versiones anteriores, ya que posibilita la definición de un grupo de workers sin instanciar ningún recurso en el _cloud provider_ que no sea necesario.
```
Comment (tperez-stratio) [inline stratio-docs/es/modules/ROOT/pages/installation.adoc:396]:
```suggestion
NOTE: Se ha implementado la opción de establecer un _min++_++size_ igual a cero, lo que permite que el autoescalado pueda incrementar o disminuir el número de nodos hasta alcanzar cero según sea necesario. Esta funcionalidad proporciona un ahorro significativo de costes en comparación con versiones anteriores ya que permite la definición de un grupo de _workers_ sin instanciar ningún recurso en el proveedor _cloud_ que no sea necesario.
```

File: stratio-docs/es/modules/ROOT/pages/installation.adoc [hunk 4]
```diff
@@ -938,7 +940,7 @@ A día de hoy estas acciones son requeridas para entornos en la nube pública _A
 
 Se trata de un operador encargado de la creación _Elastic Load Balancers_ para clústeres de _Kubernetes_ asistiendo en la creación de objetos tales como: _ingress_ y servicios de tipo _LoadBalancer_.
 
-Para garantizar una configuración adecuada de _aws-load-balancer-controller-manager_ (habilitado por defecto en entornos gestionados, EKS) es fundamental seguir una serie de pasos en los que se incluye la integración con https://docs.aws.amazon.com/es_es/eks/latest/userguide/iam-roles-for-service-accounts.html[IRSA]:
+Para garantizar una configuración adecuada de _aws-load-balancer-controller-manager_ (no habilitado por defecto en entornos gestionados, EKS) es fundamental seguir una serie de pasos en los que se incluye la integración con https://docs.aws.amazon.com/es_es/eks/latest/userguide/iam-roles-for-service-accounts.html[IRSA]:
```
Comment (tperez-stratio) [inline stratio-docs/es/modules/ROOT/pages/installation.adoc:943]:
```suggestion
Para garantizar una configuración adecuada de _aws-load-balancer-controller-manager_ (no habilitado por defecto en entornos gestionados, EKS) es fundamental seguir una serie de pasos en los que se incluye la integración con https://docs.aws.amazon.com/es_es/eks/latest/userguide/iam-roles-for-service-accounts.html[IRSA].
```

**Review by tperez-stratio — CHANGES_REQUESTED** [review]
@esierra-stratio te devuelvo un par de cambios. Por otro lado, para poder subir la versión en inglés necesito que me des permisos de escritura a tu repositorio remoto ya que ahora mismo no puedo hacer commits sobre tu PR. ¡Gracias!

**Review by tperez-stratio — APPROVED** [review]

**Pull request merged**